src/modules/services/couchdb.nix

```diff
@@ -0,0 +1,151 @@
+{ pkgs
+, lib
+, config
+, ...
+}:
+let
+  cfg = config.services.couchdb;
+  types = lib.types;
+  baseDir = config.env.DEVENV_STATE + "/couchdb";
+  configFile = pkgs.writeText "couchdb.ini" (
+    ''
+      [couchdb]
+      database_dir = ${baseDir}
+      uri_file = ${cfg.uriFile}
+      view_index_dir = ${cfg.viewIndexDir}
+      single_node = true
+
+      [admins]
+      ${cfg.adminUser} = ${cfg.adminPass}
+
+      [chttpd]
+      port = ${toString cfg.port}
+      bind_address = ${cfg.bindAddress}
+      [log]
+      file = ${cfg.logFile}
+    ''
+  );
+  startScript = pkgs.writeShellScriptBin "start-couchdb" ''
+    set -euo pipefail
+    if [[ ! -d "${baseDir}" ]]; then
+      mkdir -p "${baseDir}"
+      touch ${baseDir}/couchdb.uri
+    fi
+    
+    if ! test -e ${baseDir}/.erlang.cookie; then
+      touch ${baseDir}/.erlang.cookie
+      chmod 600 ${baseDir}/.erlang.cookie
+      dd if=/dev/random bs=16 count=1 | base64 > ${baseDir}/.erlang.cookie
+    fi
+
+    exec ${cfg.package}/bin/couchdb
+  '';
+in
+{
+  imports = [
```

Comment:
This import is not needed because this option is new, so it doesn't need renaming.